addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -266,9 +259,41 @@ $o-navbar-nav-pills-link-border-radius: $nav-pills-border-radius !default;
             }
         }
     }
-    @if $-is-body-text-light {
-        .text-muted {
-            color: color-contrast($modal-content-bg, $text-muted, mute-color($color-contrast-dark)) !important;
+    .text-muted {
+        color: adjust-color-to-background($text-muted, $modal-content-bg, mute-color($color-contrast-light), mute-color($color-contrast-dark)) !important;
+    }
+}
+
+// Popover
+.popover {
+    // The popover can have a different background color than that of the body.
+    // Here, we adjust the text color of the popover in case the body color
+    // (used by default for the text color of popovers) is not visible inside a
+    // popover due to a lack of contrast (e.g. on a website with a dark
+    // background).
+    @if $popover-header-color == null {
+        .popover-header {
+            color: adjust-color-to-background($body-color, $popover-header-bg);
+        }
+    }
+    @if $popover-body-color == $body-color {
+        .popover-body {
+            color: adjust-color-to-background($body-color, $popover-bg);
         }
     }
 }
+
+$-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
```

Comment:
It might not be a good idea to put this in a variable as it might not be used. But I thought it might be better if later we still need to fix the color of an element with a gray-200 background.